pkg/resources/resources_manager.go

```diff
@@ -178,6 +178,18 @@ func (rm *resourceManager) InitServers() error {
 		log.Printf("Resource: %+v\n", config)
 		devices := rm.GetDevices()
 		filteredDevices := rm.GetFilteredDevices(devices, &config.Selectors)
+		for _, device := range filteredDevices {
```

Comment:
Should this be the responsibility of device plugin ?

Reply:
It's only one common place of all device plugin consumers. To implement it in CNI we need to implement it in all CNIs. The same for operator: we can't fix all operators(we've got only Network Operator now but it doesn't contain daemon the each node)

Reply:
This is a workaround just to fix vmware issue. As Ivan mention solving it in the operator level now it a bit problematic.
@e0ne  maybe you can add a comment in the code on this?

Reply:
@moshe010 added a comment